ui/component/headerNotificationButton/view.jsx

```diff
@@ -1,51 +1,42 @@
 // @flow
-import * as PAGES from 'constants/pages';
+import 'scss/component/_header.scss';
+
+import { ENABLE_UI_NOTIFICATIONS } from 'config';
+import { useHistory } from 'react-router';
 import * as ICONS from 'constants/icons';
-import React from 'react';
+import * as PAGES from 'constants/pages';
+import Button from 'component/button';
 import Icon from 'component/common/icon';
 import NotificationBubble from 'component/notificationBubble';
```

Comment:
From Tom's screenshot, the bubble and stake indicator is touching borders, which is typically not good UI.
Not a show stopper, but should probably rescale accordingly if easy to do.

Reply:
So I don't know if that was an older version but I did make smaller defaults for mobile so that it wouldn't overflow out, and I noticed on his screenshot he still has the issue of the search being too far left, which was also fixed, will do some testing to find if it's related to the size of the screen

Reply:
Not sure if related or not (but maybe it does since this PR changed the envs): yesterday I updated the envs in Anthony and Salt to minimal customizations, so that it'll mostly grab whatever's in `.env.default` -- no more unexpected overrides.  The next build, if not already, should reflect this.

@tzarebczan, just fyi.  I only updated `Prod-1`, `Prod-2`, `Salt` and `Anthony`.  Left the others alone.  Also kept an original backup in their respective folders.